app/client/cli/debug.go

```diff
@@ -260,21 +260,27 @@ func fetchPeerstore(cmd *cobra.Command) (sharedP2P.Peerstore, error) {
 		logger.Global.Fatal().Msg("Unable to retrieve the bus from CLI context")
 	}
 	modulesRegistry := bus.GetModulesRegistry()
-	addrBookProvider, err := modulesRegistry.GetModule(addrbook_provider.ModuleName)
+	pstoreProvider, err := modulesRegistry.GetModule(peerstore_provider.ModuleName)
 	if err != nil {
-		logger.Global.Fatal().Msg("Unable to retrieve the addrBookProvider")
+		// CONSIDER: preferring to return errors instead of using the logger's
```

Comment:
I think I steelman both sides of the argument depending on the usecase.

In this one, I support returning an error.